extensionsGallery-insider.json

```diff
@@ -3940,6 +3940,67 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "80",
+					"extensionName": "sql-data-inspector",
+					"displayName": "SQL Data Inspector",
+					"shortDescription": "This extension simplify the inspection of data stored in tables and columns to just few clicks.",
```

Comment:
nit: is it really necessary to mention `stored in tables and columns`? especially `the columns` doesn't sound correct to me.

Reply:
we can merge the PR if you want to leave the description as is.

Reply:
Yes, thanks. I have changed the description. Now it should sound better. In the meantime I have fixed some bugs and increased the version.